docs/doxygen/Doxymain.md

```diff
@@ -0,0 +1,76 @@
+# Welcome to p4 Compiler's documentation!
```

Comment:
```suggestion
# Welcome to the documentation of the P4 compiler (P4C)!
```

Reply:
This would make the title in side bar a bit lengthy.

Reply:
I see, then I would just do `P4 Compiler Documentation`.